sdc/tests/tests_perf/test_perf_series_str.py

```diff
@@ -135,42 +73,30 @@ def _test_series_str(self, pyfunc, name, input_data=None):
             hpat_func(test_data)
 
             exec_times, boxing_times = get_times(hpat_func, test_data, iter_number=self.iter_number)
-            self.test_results.add(name, 'JIT', test_data.size, exec_times, data_width,
+            self.test_results.add(test_name, 'JIT', test_data.size, exec_times, data_width,
                                   boxing_times, compile_results=compile_results, num_threads=self.num_threads)
             exec_times, _ = get_times(pyfunc, test_data, iter_number=self.iter_number)
-            self.test_results.add(name, 'Reference', test_data.size, exec_times, data_width,
+            self.test_results.add(test_name, 'Reference', test_data.size, exec_times, data_width,
                                   num_threads=self.num_threads)
 
-    def test_series_str_len(self):
-        self._test_series_str(usecase_series_len, 'series_str_len')
-
-    def test_series_str_capitalize(self):
-        self._test_series_str(usecase_series_capitalize, 'series_str_capitalize')
-
-    def test_series_str_lower(self):
-        self._test_series_str(usecase_series_lower, 'series_str_lower')
-
-    def test_series_str_swapcase(self):
-        self._test_series_str(usecase_series_swapcase, 'series_str_swapcase')
-
-    def test_series_str_title(self):
-        self._test_series_str(usecase_series_title, 'series_str_title')
-
-    def test_series_str_upper(self):
-        self._test_series_str(usecase_series_upper, 'series_str_upper')
-
-    def test_series_str_lstrip(self):
-        input_data = ['\t{}  '.format(case) for case in test_global_input_data_unicode_kind4]
-        self._test_series_str(usecase_series_lstrip, 'series_str_lstrip', input_data=input_data)
-
-    def test_series_str_rstrip(self):
-        input_data = ['\t{}  '.format(case) for case in test_global_input_data_unicode_kind4]
-        self._test_series_str(usecase_series_rstrip, 'series_str_rstrip', input_data=input_data)
-
-    def test_series_str_strip(self):
-        input_data = ['\t{}  '.format(case) for case in test_global_input_data_unicode_kind4]
-        self._test_series_str(usecase_series_strip, 'series_str_strip', input_data=input_data)
-
 
-if __name__ == "__main__":
-    unittest.main()
+cases = [
```

Comment:
Please, add short description for each parameter in the tuple. So it will be easier for people to read and support it, and also add new test cases.